### DISCUSSION TITLE
Ruler options: acc, progress

### DISCUSSION THREAD

**Discussion by og900aero** [gokcehan/lf#1218] [Q&A] [answered]
In r29 there is ruler options. But when I enabled acc and progress, I doesnt see aníthing. Selection, ind and df working great. What is acc and progress value?

**Comment by gokcehan** [comment, marked as answer]
@og900aero Acc and progress only exists at certain times. Acc shows you the pressed keys (e.g. for bindings with multiple key presses or counts given to bindings) and progress shows the progress of file operations. We should probably explain the field names in the documentation.

**Reply by og900aero** [reply]
> @og900aero Acc and progress only exists at certain times. Acc shows you the pressed keys (e.g. for bindings with multiple key presses or counts given to bindings) and progress shows the progress of file operations. We should probably explain the field names in the documentation.

Acc right, works well. But progress...I try copy directory, but doesnt show progress.

**Reply by joelim-work** [reply]
Try copying something really big. The progress field only shows when something actually is in progress.

**Reply by og900aero** [reply]
> @og900aero Acc and progress only exists at certain times. Acc shows you the pressed keys (e.g. for bindings with multiple key presses or counts given to bindings) and progress shows the progress of file operations. We should probably explain the field names in the documentation.

Ohh my god! The progress value is so fantastic and working great! No more custom copy program (like advcp, advmv). Thank u so much dear gokcehan!
